_data/simple-icons.json

```diff
@@ -8010,7 +8014,7 @@
         {
             "title": "Scrum Alliance",
             "hex": "009FDA",
-            "source": "https://www.scrumalliance.org/ScrumRedesignDEVSite/media/ScrumAllianceMedia/Files%20and%20PDFs/Infographics/S_BrandGuidelines_2018_rev.pdf",
+            "source": "https://scrum.mx/advanced-certified-scrum-master",
```

Comment:
I'm not sure for this one since there are no vector version available there

Reply:
It's probably from [the resources page](https://resources.scrumalliance.org/), still not ideal though.

More broadly, I do think that the fact that @jorgeamadosoria's suggested change is neither an SVG nor an "official" source (and arguably also that it's not English(*)) discredits it. I don't think it makes sense to replace an "incorrect" (by our own standards) source with another "incorrect" one makes sense.

(*): not discrediting Spanish sources in general, but the primary language of this project is English - so if there are English sources, they should be preferred.